pages/osx/uname.md

```diff
@@ -3,18 +3,22 @@
 > Print details about the current machine and the operating system running on it.
 > Note: for additional information about the operating system, try the `sw_vers` command.
```

Comment:
needs a more info link, the GNU one might be applicable

Reply:
How about <https://ss64.com/osx/uname.html>?

Reply:
Considering that other pages have this website, I don't see a problem. However I am still pretty new here and don't know all the rules and things that have changed.

cc @navarroaxel for your opinions since I don't know whether to apply this

Reply:
@CleanMachine1 the long arguments are not available in macOS, only the flags listed in the ss64 website. I checked this in macOS Big Sur

Reply:
Oh, ok.

I'll fix them

Reply:
https://github.com/tldr-pages/tldr/pull/6242/commits/83de6d2424abf6a7b6a47d4f1827908acc2b1f46